### PR TITLE
feat: add some metrics about RT

### PR DESCRIPTION
**Pull request by ev1lQuark**
The minimum response time (min rt) metrics for different labels are maintained using `sync.Map`. The purpose of labels is to differentiate between different provider-consumer RPC invocation paths. For example, if provider A offers a service and is called by two consumers, Consumer A and Consumer B, the minimum response time for each path needs to be tracked separately.

Please provide suggestions for the code organization, naming, and implementation approach for this PR.

---

使用` sync.Map `对于不同 label 的最小响应时间（min rt）指标进行维护。label 的作用是区分不同的 provider-consumer RPC 调用链路，比如 provider A 提供的服务被 consumer A 和 Consumer B 两个消费者去调用，则两条链路的最小响应时间需要分别统计。

请对于本 PR 代码组织方式、命名、实现思路提出建议。

### CODE REVIEW

File: metrics/prometheus/util.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	}, "_")
+}
+
+func updateMin(m *sync.Map, labels *prometheus.Labels, gaugeVec *prometheus.GaugeVec, costMs int64) {
+	key := convertLabelsToMapKey(*labels)
+	for {
+		if actual, loaded := m.LoadOrStore(key, costMs); loaded {
+			if costMs < actual.(int64) {
+				// need to update
+				if m.CompareAndSwap(key, actual, costMs) {
```
Comment (FoghostCn) [inline metrics/prometheus/util.go:47]: CompareAndSwap 是go1.20 加入的 api

Comment (ev1lQuark) [reply metrics/prometheus/util.go]: 感谢指正，新的提交使用了更原始的方式，请检查并发安全

**Comment by ev1lQuark** [comment]
1. Add more metrics:
    - dubbo_xxx_rt_milliseconds_max
    - dubbo_xxx_rt_milliseconds_avg
    - dubbo_xxx_rt_milliseconds_sum
    - dubbo_xxx_rt_milliseconds_last
2. refactor some file names and method names

**Review by chickenlj — APPROVED** [review]
LGTM.

**Review by justxuewei — APPROVED** [review]
Lgtm, thanks!

**Pull request merged**